app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailer < ApplicationMailer
+
+  def account_activation user
+    @user = user
+    mail to: user.email, subject: t ".subject"
```

Comment:
I think you should add () to i18n of subject 
`mail to: user.email, subject: t(".subject")`